pg_analytics/Cargo.toml

```diff
@@ -23,7 +23,7 @@ pg_test = []
 telemetry = ["shared/telemetry"]
 
 [dependencies]
-pgrx = { git = "https://github.com/pgcentralfoundation/pgrx.git", rev = "bef7ab7" }
+pgrx = { git = "https://github.com/paradedb/pgrx.git", branch = "rebasedming/rmgr" }
```

Comment:
Does this cause any incompatibility issues with `cargo-pgrx`?

Reply:
my PR to pgrx just got merged! will swap back to upstream